docs/override-configs.md

```diff
@@ -1,3 +1,29 @@
+## Enabling/disabling components
+
+To enable/disable components in Ozone, you need to modify the configurations in the [docker-compose-files.txt](https://github.com/ozone-his/ozone-docker-compose/blob/main/scripts/docker-compose-files.txt) in the ozone-docker-compose for your Ozone deployment. Here's how you can do it:
+
+**Access the excutable files**
+
+Navigate to the docker-compose-files.txt file in the ozone-docker-compose.
+
+**Enable/disable a component**
+
+To enable a component, you need to provide it's excutable file in the ozone-docker-compose.
```

Comment:
This in not clear, or not true.

Something such as this is more accurate:
```suggestion
To enable a component, you need to add the name of the corresponding Docker Compose file in the docker-compose-files.txt.
```

Could you provide an example here?